site/src/modules/workspaces/WorkspaceTiming/ResourcesChart.tsx

```diff
@@ -89,7 +84,7 @@ export const ResourcesChart: FC<ResourcesChartProps> = ({
 			<ChartContent>
 				<YAxis>
 					<YAxisSection>
-						<YAxisHeader>{stage} stage</YAxisHeader>
+						<YAxisHeader>{stage.name} stage</YAxisHeader>
```

Comment:
In your screenshot, this section will be named `main` (or whatever the agent resource is named in the template); I don't think this will make sense to users.

In another PR I suggested that a single agent should show "agent", and multiple agents (uncommon case) should show "agent ($name)".

Reply:
I don't think we should do this. It looks confusing to me display the same agent data differently depending on how many agents are present. Wdyt?

Reply:
Maybe we should always show `agent ($name)` then?
Just showing `$name` will be confusing. Folks might not name their agent resource very clearly.